addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/ChannelTypeUtils.java

```diff
@@ -71,6 +71,10 @@ public static State toDecimalType(@Nullable BigDecimal decimal) {
         return decimal == null ? UnDefType.NULL : new DecimalType(decimal.setScale(2, BigDecimal.ROUND_HALF_UP));
     }
 
+    public static State toDecimalType(@Nullable String textualDecimal) {
+        return textualDecimal == null ? UnDefType.NULL : new DecimalType(textualDecimal);
```

Comment:
In the light of https://github.com/eclipse/smarthome/issues/4612#issuecomment-354744934, shouldn't we rather completely remove the use of `UnDefType.NULL` here?

Reply:
I'm able to fix the bug (and make it consistent with the rest of the implementation) for this user but I cannot spent the time to rebuild the netatmo to not send updates for `null` values, that would require a bigger refactoring.

Maybe @clinique does have time for this.